backends-velox/src/main/scala/org/apache/gluten/execution/datasource/v2/ArrowBatchScanExec.scala

```diff
@@ -31,6 +33,10 @@ case class ArrowBatchScanExec(original: BatchScanExec)
 
   @transient lazy val batch: Batch = original.batch
 
+  override protected def batchType0(): Convention.BatchType = {
```

Comment:
nit: the name batchType0 is a bit confusing, and can we use a more informative name?